gatus.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: gatus
+  version: 5.15.0
+  epoch: 0
+  description: Automated developer-oriented status page
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 8aae277113753a67a324e9658c261eb96e563556
+      repository: https://github.com/TwiN/gatus.git
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: -X main.version=v${{package.version}}
```

Comment:
```suggestion

```
this can be omitted given upstream doesn't have this variable configured. (https://github.com/TwiN/gatus/blob/master/main.go)